client/index.html

```diff
@@ -15,6 +15,15 @@
 	<link rel="stylesheet" href="css/bootstrap.css">
 	<link rel="stylesheet" href="css/style.css">
 	<link id="theme" rel="stylesheet" href="<%= theme %>">
+	<style id="user-specified-css">/* Easily modify font settings used in chat window */
```

Comment:
Set `class="input"` here and you will be able to remove some of the CSS while integrating better with the other form elements. Dumb example [here](https://dl.dropboxusercontent.com/u/3400500/mockup_thelounge_settings/themes.html).